### ISSUE TITLE
Running sythentic training set

### ISSUE THREAD

**Issue by robomotic** [gerstung-lab/Delphi#12]
Hi there,
amazing work, I have start already looking at the code:
`python train.py config/train_delphi_demo.py --device=cuda --out_dir=Delphi-2M`
And the last iteration:
```
saving checkpoint to Delphi-2M
iter 5000: loss 10.5874, time 1258.89ms
```
Isn't that too high of a loss for training?

**Comment by yozhikoff** [comment]
Thanks!

It's what it should be, more or less. 
But please be aware that the provided synthetic dataset is extremely small. It allows checking that the code runs as well as getting familiar with the format, but one shouldn't expect to be able to reproduce the results in the paper using it.
